client/src/components/StepByStep.js

```diff
@@ -18,33 +18,29 @@ Step.propTypes = {
 };
 
 const StepByStep = () => (
-  <Col xs={10} xsOffset={1} md={6} mdOffset={3} className="home-section-container">
-    <Row>
-      <Col xs={12}>
-        <PageHeader className="home-section-page-header">How does this work?</PageHeader>
-      </Col>
-      <Step
-        emojiIcon="&#x270D;"
-        title="Create a campaign"
-        description="Create and share your Multi-family Complex (MFC's) campaign page with your neighbors."
-      />
-      <Step
-        emojiIcon="&#x1F4CB;"
-        title="Recruit support"
-        description="Voice and organize your support."
-      />
-      <Step
-        emojiIcon="&#x1F64F;"
-        title="Request"
-        description="Request recycling services from your landlord by sharing your building's campaign page."
-      />
-      <Step
-        emojiIcon="&#x1F64C;"
-        title="Recycle!"
-        description="Recycle and live in a more sustainable building."
-      />
-    </Row>
-  </Col>
+  <div className="StepByStep">
```

Comment:
** [class-wrapper-names] ** We have been trying to use a wrapper syntax for these types of classes. Should we consider that here too?